Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use a minimal base image with NVIDIA GPU support
+FROM livepeer/go-livepeer:0.7.1
+
+ENV	NVIDIA_DRIVER_CAPABILITIES=all
+
+# Install required packages
+RUN apt-get update && apt-get install -y \
+    wget \
+    tar \
+    psmisc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download and extract the Titan Node Pool binary
+RUN wget https://files.titan-node.com/Titan_Node_Pool_v1.34_Linux.tar.gz && \
```

Comment:
An enhancement suggestion: Consider leveraging GitHub releases alongside the `latest` label, complemented by a GitHub action that automatically affixes this label to the most recent release. This streamlined approach ensures that users can effortlessly access the latest version.

Reply:
Moreover, given your access to the binary source code, you can consider employing a multi-layer build. This approach entails constructing the binary in a dedicated build step and transferring it to a final image, ready for publication on Docker Hub. This will allow faster deployment compared to fetching the `tar.gz` locally ⚡.